rgw/v2/tests/s3_swift/test_bucket_lifecycle_object_expiration.py

```diff
@@ -61,9 +61,12 @@ def test_exec(config):
     ceph_conf.set_to_ceph_conf('global', ConfigOpts.rgw_lc_debug_interval, str(config.rgw_lc_debug_interval))
     ceph_version = utils.exec_shell_cmd("ceph version")
     op = ceph_version.split()
-    for i in op:
-        if i in ['nautilus','pacific']:
-                ceph_conf.set_to_ceph_conf('global', ConfigOpts.rgw_lc_max_worker, str(config.rgw_lc_max_worker))
+    if "pacific" in op:
+        commands = [f"ceph config set client.rgw rgw_lc_max_worker {config.rgw_lc_max_worker}",
+                                    "ceph config set client.rgw rgw_lc_debug_interval 30"]
+        for command in commands: utils.exec_shell_cmd(command)
+    if 'nautilus' in op:
```

Comment:
do you want to add a condition for `luminous`. Though its EOL, we should should still support the tests for some time.

Reply:
We don't need to set any parameter for Luminous. rgw_lc_debug_interval would be set for all the releases (Line 61).